BREAKING.md

```diff
@@ -1,10 +1,16 @@
 ## 0.43 Breaking changes
 
 - [TinyliciousClient no longer static](#TinyliciousClient-no-longer-static)
```

Comment:
I don't think we make any update here.  0.43 client release isn't changing in any way with this change.

Reply:
Right, put a note here too since most people look at the root BREAKING.md, but I can remove it. (Or are you saying pin it to the top or something)

Reply:
Ok I see what you're getting at.  Nonetheless, having this in here will be the wrong time/place for anyone to be aware - it's totally arbitrary relative to when the change will actually hit anyone.  Perhaps when we bump the client's dependency on the defs packages, we'll include a note in this BREAKING.md saying to look at all the def's breaking changes or something?

cc @anthony-murphy